pkg/kubelet/util/util.go

```diff
@@ -30,6 +31,8 @@ func FromApiserverCache(opts *metav1.GetOptions) {
 }
 
 func parseEndpoint(endpoint string) (string, string, error) {
+	// url.Parse doesn't recognize \, so replace with / first.
+	endpoint = strings.Replace(endpoint, "\\", "/", -1)
```

Comment:
I don't think it's safe to do this on non-windows platforms.

Either use libraries that can handle paths differently on different OS, or rewrite this function in `_windows.go`
The latter might be good since protocols like `unix://` cannot be accepted on windows.

Reply:
Good catch. Thanks.